include/swift/Parse/Token.h

```diff
@@ -159,6 +159,10 @@ class Token {
     return isAnyOperator() && Text == "~";
   }
 
+  bool isMinus() const {
+    return isAnyOperator() && Text == "-";
+  }
```

Comment:
We could probably also use this in `ParseExpr.cpp`'s `Parser::parseExprUnary` line 690 where we check for a minus sign before an integer literal.